schema/test/unit/migrations/008_replace_bad_field_capitalization_test.sql

```diff
@@ -0,0 +1,170 @@
+begin;
+
+select * from plan(4);
+
+/** SETUP */
+
+truncate cif.project restart identity cascade;
+
+-- create three projects
+select cif.create_project(1); -- this creates project 1 with form_change.id=1
+select cif.create_project(1); -- this creates project 2 with form_change.id=2
+
+-- project 1, milestone 1
+select cif.create_form_change(
+        'create',
+        'milestone',
+        'cif',
+        'reporting_requirement',
+        json_build_object(
+          'projectId', 1,
+          'reportDueDate', '2000-12-12 16:21:42.693489-07',
+          'submittedDate', '2000-12-12 16:21:42.693489-07',
+          'reportType', 'General Milestone',
+          'reportingRequirementIndex', 1,
+          'description', 'general milestone report description ' ,
+          'adjustedNetAmount', 4,
+          'adjustedGrossAmount', 5,
+          'adjustedHoldBackAmount', 1,
+          'dateSentToCsnr', '2022-03-01 16:21:42.693489-07',
+          'certifierProfessionalDesignation', 'Professional Engineer',
+          'substantialCompletionDate', '2000-12-12 16:21:42.693489-07',
+          'maximumAmount', 555,
+          'totalEligibleExpenses', 1000,
+          'certifiedBy', 'Elliot Page',
+          'hasExpenses', true
+        )::jsonb,
+        null,
+        1
+      );
+
+-- project 1, milestone 2, 2021/2022 (date sent to csnr), adjusted gross
+select cif.create_form_change(
+        'create',
+        'milestone',
+        'cif',
+        'reporting_requirement',
+        json_build_object(
+          'projectId', 1,
+          'reportDueDate', '2000-12-12 16:21:42.693489-07',
+          'submittedDate', '2000-12-12 16:21:42.693489-07',
+          'reportType', 'General Milestone',
+          'reportingRequirementIndex', 2,
+          'description', 'general milestone report description ' ,
+          'adjustedGrossAmount', 10,
+          'adjustedNetAmount', 9,
+          'adjustedHoldBackAmount', 1,
```

Comment:
have you tested this on dummy data that has an attribute called `adjustedHoldbackAmount` to confirm there are no adverse effects on existing data with the correct capitalization?

Reply:
There won't be any in the data, all current records have the bad capitalization. But you reminded me that I should probably add another test for idempotency (which will also cover the scenario you brought up)